.coverage

```diff
@@ -0,0 +1 @@
oleo-Dojo-Project/tests/test_load_state.py":[1,2,4,7,18,19,20,21,36,43,47,51,55,59,63,66,22,23,24,26,27,29,30,31,32,33,64,48,44,56,67,37,38,52,60],"/Users/georgreen/Projects/Geoogreen-Mamboleo-Dojo-Project/models/model.py":[1,2,3],"/Users/georgreen/Projects/Geoogreen-Mamboleo-Dojo-Project/models/room.py":[2,5,7,8,10,21,28,34,40,46,55,63,67,69,70,72,78,82,84,85,87,73,74,14,15,16,17,18,19,88,89,44,38,32,26],"/Users/georgreen/Projects/Geoogreen-Mamboleo-Dojo-Project/models/facility.py":[1,5,6,8,30,40,43,46,51,56,60,64,70,76,79,82,88,99,108,117,127,139,140,9,11,13,31,32,33,35,37,38,14,17,18,19,20,21,22,23,24,25,26,27,58,62,53,54,48,49,83,84,85,86,77,80,89,90,91,92,93,97,41,94,95,96,10,12,74,34,15,122,125,112,113,114,103,104,105,132,133,134],"/Users/georgreen/Projects/Geoogreen-Mamboleo-Dojo-Project/models/person.py":[1,3,5,11,18,21,25,27,29,35,38,42,45,51,53,55,59,30,31,6,7,8,9,32,33,56,57,16,36,22,19],"/Users/georgreen/.virtualenvs/amityenv/lib/python3.6/site-packages/nose/case.py":[30,34,35,37,38,39,40,41,42,43,100,101,102,103,104,105,46,129,130,131,132,133,70,71,75,134,148,149,150,152,141,60,61,65,155,156,160,161,162,163,164,167,168,169,173,174,179,180,186,49,50,52],"/Users/georgreen/Projects/Geoogreen-Mamboleo-Dojo-Project/tests/test_logic.py":[1,3,4,7,8,9,14,18,22,26,29,32,37,52,53,58,68,73,87,88,120,126,130,134,135,164,172,173,198,54,55,56,69,70,71,59,60,61,62,63,64,65,66,10,11,27,23,24,19,20,15,16,30,33,136,137,138,139,141,142,143,145,146,147,148,149,150,151,152,153,154,156,157,158,160,161,162,166,174,175,176,177,178,179,181,182,184,185,186,187,188,190,191,193,194,195,196,200,201,202,203,204,205],"/Users/georgreen/Projects/Geoogreen-Mamboleo-Dojo-Project/core/logic.py":[1,2,3,6,34,65,91,104,105,108,121,144,148,171,230,242,127,128,131,132,133,134,135,138,141,173,174,175,176,182,183,188,189,243,244,245,248,249,252,253,254,190,194,195,198,199,200,201,203,205,206,208,231,232,234,236,239,209,210,211,213,214,215,216,217,218,222,227,202,219,223],"/Users/georgreen/Projects/Geoogreen-Mamboleo-Dojo-Project/core/helpers.py":[1,2,5,35,72,91,112,124,136,137,140,150,162,173,40,41,45,46,50,51,52,54,57,58,60,62,63,64,69,66,67,68,53,42,14,15,17,19,21,23,26,28,30,32,22,29,77,80,81,82,83,84,88],"/Users/georgreen/Projects/Geoogreen-Mamboleo-Dojo-Project/tests/test_model.py":[1,2,3,4,7,18,19,25,30,33,38,41,44,48,52,66,67,90,94,102,106,109,112,115,118,122,125,129,146,147,172,179,189,194,198,203,208,213,218,223,20,21,22,23,26,27,31,34,35,36,42,39,49,45,46,148,149,151,152,154,155,157,158,160,161,164,165,166,167,168,169,173,174,175,176,190,191,204,205,195,199,200,224,225,214,215,209,210,219,220,180,181,182,183,68,69,70,72,73,75,76,78,79,81,82,84,85,87,88,95,96,97,99,100,116,119,120,126,107,123,110,113,91,92,103,104],"/Users/georgreen/Projects/Geoogreen-Mamboleo-Dojo-Project/tests/context.py":[3,4,5,7,8,9],"/Users/georgreen/Projects/Geoogreen-Mamboleo-Dojo-Project/tests/test_save_state.py":[2,3,5,8,16,17,18,30,34,38,42,49,19,20,21,22,23,25,26,27,28,35,31,50,43,44,39]}}
```

Comment:
@andela-jkanyua The build is failing because we have to implement failing tests first then afterward implement the functionality which will make them pass, we have been instructed to follow the TDD development process.